src/fileoverview_comment_transformer.ts

```diff
@@ -0,0 +1,80 @@
+import * as ts from 'typescript';
+import * as jsdoc from './jsdoc';
+
+const FILEOVERVIEW_COMMENTS: ReadonlySet<string> =
```

Comment:
Can you doc?  I think these are the comments that mark a jsdoc block as a fileoverview comment?  Or are they the ones allowed in a block that must have fileoverview?

Reply:
done.